android/scripts/gfxrecon.py

```diff
@@ -134,6 +135,9 @@ def MakeExtrasString(args):
     if args.onhb:
         arg_list.append('--onhb')
 
+    if args.virtual_swapchain:
+        arg_list.append('--virtual-swapchain')
```

Comment:
I think this needs to be updated to `--use-captured-swapchain-indices`.

Reply:
Fixed. Thanks.